src/mann_whitney.jl

```diff
@@ -52,7 +71,17 @@ end
 
 
 ## EXACT MANN-WHITNEY U TEST
+"""
+    ExactMannWhitneyUTest(x::AbstractVector{T<:Real}, y::AbstractVector{T<:Real})
 
+Perform an exact Mann-Whitney U test.
```

Comment:
Copy the definition of this test from the above docstring, as it should be usable in isolation. Same for the approximate test.